src/index.html

```diff
@@ -6,9 +6,54 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="home" class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="logo image"
+          class="image__logo"
+          >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="list-item">
+            <a href="#apple" class="link-item is-active">Apple</a>
+          </li>
+          <li class="list-item">
+            <a href="#samsung" class="link-item">Samsung</a>
+          </li>
```

Comment:
Extra line here would make it more readable :) (extra lines between children in general)
```suggestion
          <li class="list-item">
            <a href="#apple" class="link-item is-active">Apple</a>
          </li>
          
          <li class="list-item">
            <a href="#samsung" class="link-item">Samsung</a>
          </li>
```